docs/tools/CLI/cli-create.md

```diff
@@ -108,9 +108,59 @@ $ mbed compile -t ARM -m K64F --source project2 --source mbed-os --build BUILD/p
 
 Find more details on the `--source` switch in the [build rules documentation](../reference/mbed-os-build-rules.html).
 
-### Updating to an upstream version
+### Updating programs and libraries
 
-#### Updating a program
+You can update programs and libraries on your local machine so that they pull in changes from the remote sources (Git or Mercurial).
```

Comment:
"pull in changes" sounds very colloquial, and may not mean much in this context. Perhaps "update to the latest released version" would be a better replacement.

Reply:
Done